data/locales.json

```diff
@@ -34,13 +34,6 @@
         "footer_tagline": "Simply the best portfolio management tool for DIY investors.",
         "sitemap_priority": 0.8
       },
-      {
-        "page": "survey-thanks",
-        "page_title": "Thanks | Sharesight",
-        "page_description": "Thanks for taking the time to complete our survey.",
-        "footer_tagline": "Simply the best portfolio management tool for DIY investors.",
-        "no_index": true
-      },
```

Comment:
Oh, I didn't think about this in the other review, two comments:
1. This page is only available globally, not in locales, eg. https://www.sharesight.com/nz/survey-thanks 404s.  I don't think this matters too much; if in Gatsby `/nz/survey-thanks` does not 404, that's probably fine.
2. This is `no_index: true`.  Is the new SurveyThanks page you built that too?

Reply:
ooh good point, will make a note of double checking the locales and yup the config in contentful has been set up such that disable indexing = yes